benches/benchmark_results.txt

```diff
@@ -0,0 +1,8 @@
+Latency: 7.013267021 seconds
```

Comment:
Criterion already calculate execution time. Explain why we need the other stats and what they are.
Maybe we should use other tools for that like iai https://github.com/bheisler/iai.